bundle/edu.gemini.pot/src/test/scala/edu/gemini/spModel/target/env/Arbitraries.scala

```diff
@@ -54,15 +55,15 @@ trait Arbitraries extends edu.gemini.spModel.core.Arbitraries {
   implicit def arbNonEmptyList[A: Arbitrary]: Arbitrary[NonEmptyList[A]] =
     Arbitrary {
       for {
-        a  <- arbitrary[A]
+        a <- arbitrary[A]
```

Comment:
I didn't mean to misalign the arrows. I blame IDEA.

Reply:
Yeah, IDEA does that